vagrant/Vagrantfile

```diff
@@ -0,0 +1,139 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'socket'
+
+hostname = Socket.gethostname
+localmachineip = IPSocket.getaddress(Socket.gethostname)
+puts %Q{ This machine has the IP '#{localmachineip} and host name '#{hostname}'}
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = '2'
+
+# Validate required plugins
+REQUIRED_PLUGINS = %w(vagrant-registration vagrant-hostmanager landrush)
+errors = []
+
+def message(name)
+  "#{name} plugin is not installed, run `vagrant plugin install #{name}` to install it."
+end
+# Validate and collect error message if plugin is not installed
+REQUIRED_PLUGINS.each { |plugin| errors << message(plugin) unless Vagrant.has_plugin?(plugin) }
+unless errors.empty?
+  msg = errors.size > 1 ? "Errors: \n* #{errors.join("\n* ")}" : "Error: #{errors.first}"
+  fail Vagrant::Errors::VagrantError.new, msg
+end
+
+if ENV['DEPLOYMENT_TYPE'] == 'enterprise'
+  box_name = 'rhel/7.2'
+else
+  box_name = 'centos/7'
+end
+
+NETWORK_BASE = '192.168.50'
+INTEGRATION_START_SEGMENT = 20
+
+$miscellany = <<SCRIPT
+echo "redhat" | sudo passwd root --stdin
+sudo sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/' /etc/ssh/sshd_config
+sudo service sshd restart
+
+#sudo yum clean all ; yum repolist
+#sudo yum -y update
+sudo yum -y install wget git net-tools bind-utils iptables-services bridge-utils bash-completion
+
+# Vagrant's "change host name" capability for Fedora
+# maps hostname to loopback, conflicting with hostmanager.
+# We must repair /etc/hosts
+
+sudo sed -ri 's/127\.0\.0\.1\s.*/127.0.0.1 localhost localhost.localdomain/' /etc/hosts
+SCRIPT
```

Comment:
This looks static, any reason not to have this as just a file alongside the Vagrantfile?

Also, is there an issue with the ssh key injection with Vagrant? We shouldn't need the root password.